var/spack/repos/builtin/packages/opa-psm2/package.py

```diff
@@ -20,16 +21,27 @@ class OpaPsm2(MakefilePackage):
     version('10.2-235', '23539f725a597bf2d35aac47a793a37b')
     version('10.2-175', 'c542b8641ad573f08f61d0a6a70f4013')
 
+    variant('avx2', default=True, description='Enable AVX2 instructions')
+
     depends_on('numactl')
 
     def setup_environment(self, spack_env, run_env):
         spack_env.set('DESTDIR', self.prefix)
         run_env.prepend_path('CPATH',
-                             join_path(self.prefix, 'usr', 'include'))
+                             join_path(self.prefix, 'include'))
         run_env.prepend_path('LIBRARY_PATH',
-                             join_path(self.prefix, 'usr', 'lib64'))
+                             join_path(self.prefix, 'lib'))
         run_env.prepend_path('LD_LIBRARY_PATH',
-                             join_path(self.prefix, 'usr', 'lib64'))
+                             join_path(self.prefix, 'lib'))
```

Comment:
You can remove these last 3 settings now that they are in the correct directory. Spack automatically adds these to the module files already.